src/DeviceHardDrive.php

```diff
@@ -233,7 +248,8 @@ public function getImportCriteria()
 
         return ['designation'       => 'equal',
             'manufacturers_id'  => 'equal',
-            'interfacetypes_id' => 'equal'
+            'interfacetypes_id' => 'equal',
+            'deviceharddrivetypes_id' => 'equal'
```

Comment:
```deviceharddrivetypes_id``` must not be added as a reconciliation key

Otherwise, the native inventory will create new harddrive (see the changes added in Unit Tests).